File: onyx-database/src/main/java/com/onyx/stream/QueryMapStream.java
package com.onyx.stream;

/**
 * Created by tosborn1 on 5/19/16.
 *
 * This is a custom lambda for the stream api.  It will indicate the expected
 * value is a map rather than hydrated entities
 */
@FunctionalInterface
public interface QueryMapStream extends QueryStream {

}

File: onyx-database/src/main/kotlin/com/onyx/persistence/stream/QueryMapStream.kt
package com.onyx.persistence.stream

/**
 * Created by Tim Osborn on 5/19/16.
 *
 * This is a custom lambda for the stream api.  It will indicate the expected
 * value is a map rather than hydrated entities
 */
interface QueryMapStream<in T : Map<String, Any?>> : QueryStream<T>
